apps/iris/main.go

```diff
@@ -73,6 +54,24 @@ func main() {
 		}()
 	}
 
+	if env == Production || env == Stage || utils.Getenv("ENABLE_OPENTELEMETRY", "false") == "true" {
```

Comment:
env용 변수는 따로 두는 거 어떨까요?

```suggestion
	enableOtel = utils.Getenv("ENABLE_OPENTELEMETRY", "false") == "true"
	if env == Production || env == Stage || enableOtel {
```